infrastructure/cdn-in-a-box/traffic_stats/datasources.yml.template

```diff
@@ -0,0 +1,81 @@
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# list of datasources to insert/update depending what's available in the database
+datasources:
+  # <string, required> name of the datasource. Required
+- name: cache_stats
+  # <string, required> datasource type. Required
+  type: influxdb
+  # <string, required> access mode. proxy or direct (Server or Browser in the UI). Required
+  access: proxy
+  # <int> org id. will default to orgId 1 if not specified
+  orgId: 1
+  # <string> url
+  url: http://$INFLUXDB_HOST:$INFLUXDB_PORT
+  # <string> database password, if used
+  password: $INFLUXDB_ADMIN_PASSWORD
+  # <string> database user, if used
+  user: $INFLUXDB_ADMIN_USER
+  # <string> database name, if used
+  database: cache_stats
+  # <bool> enable/disable basic auth
+  basicAuth: false
+  # <bool> mark as default datasource. Max one per org
+  isDefault: true
+
+  # <string, required> name of the datasource. Required
+- name: daily_stats
+  # <string, required> datasource type. Required
+  type: influxdb
+  # <string, required> access mode. proxy or direct (Server or Browser in the UI). Required
+  access: proxy
+  # <int> org id. will default to orgId 1 if not specified
+  orgId: 1
+  # <string> url
+  url: http://$INFLUXDB_HOST:$INFLUXDB_PORT
+  # <string> database password, if used
+  password: $INFLUXDB_ADMIN_PASSWORD
+  # <string> database user, if used
+  user: $INFLUXDB_ADMIN_USER
+  # <string> database name, if used
+  database: daily_stats
+  # <bool> enable/disable basic auth
+  basicAuth: false
+  # <bool> mark as default datasource. Max one per org
+  isDefault: false
+
+  # <string, required> name of the datasource. Required
+- name: deliveryservice_stats
+  # <string, required> datasource type. Required
+  type: influxdb
+  # <string, required> access mode. proxy or direct (Server or Browser in the UI). Required
+  access: proxy
+  # <int> org id. will default to orgId 1 if not specified
+  orgId: 1
+  # <string> url
+  url: http://$INFLUXDB_HOST:$INFLUXDB_PORT
+  # <string> database password, if used
+  password: $INFLUXDB_ADMIN_PASSWORD
+  # <string> database user, if used
+  user: $INFLUXDB_ADMIN_USER
+  # <string> database name, if used
+  database: deliveryservice_stats
+  # <bool> enable/disable basic auth
+  basicAuth: false
+  # <bool> mark as default datasource. Max one per org
+  isDefault: false
```

Comment:
I could say it basically copied from [here](http://docs.grafana.org/administration/provisioning/#example-datasource-config-file)
Grafana is under [apache license](https://github.com/grafana/grafana/blob/master/LICENSE.md) and I have removed unnecessary variables.
Do you mean all the variable description should be removed to avoid license problem?

Reply:
I'm not actually sure, we'll need a third opinion on this, probably from #dev in the Slack workspace